templates/style/style.scss

```diff
@@ -280,7 +280,7 @@ div.recent-releases-container {
         background-color: var(--background-color);
     }
 
-    .release {
+    .release, .build-in-progress {
```

Comment:
About the new HTML/CSS changes: it'd be nice to have a GUI test for that. :)

Reply:
I'm not sure how that would look like